internal/httpcore/endpoints_elections.go

```diff
@@ -429,3 +437,48 @@ func (endpoints) apiWithdrawFromElection(ctx *fiber.Ctx) error {
 	ctx.Status(fiber.StatusNoContent)
 	return nil
 }
+
+func (endpoints) apiGetElectionOutcome(ctx *fiber.Ctx) error {
+	userID, authStatus := getSessionAuth(ctx)
+	if authStatus == authNotAuthed {
+		return fiber.ErrUnauthorized
+	}
+
+	electionID := ctx.QueryInt("election_id")
```

Comment:
Instead of `election_id`, I would be tempted to use `id` to keep consistency with the rest of the codebase where only one type of ID is involved, eg. in `apiAdminStartElection`